app/data/Owners.scala

```diff
@@ -27,46 +27,55 @@ trait Owners {
 
 object Owners extends Owners {
 
-  override def default: Owner = Owner("dig.dev.tooling")
+  override def default: Owner = Owner("devx.sec.ops")
 
   override def stacks: Set[(String, SSA)] = Set(
-    "security.dev" -> SSA("security"),
+    "devx.sec.ops" -> SSA("security"),
+    "devx.sec.ops" -> SSA("deploy"),
+    "dotcom.platform" -> SSA(stack = "abacus"),
+    "dotcom.platform" -> SSA("domains"),
     "dotcom.platform" -> SSA("frontend"),
     "dotcom.platform" -> SSA("frontend-elk"),
     "identitydev" -> SSA(stack = "discussion"),
-    "dotcom.platform" -> SSA(stack = "abacus"),
     "identitydev" -> SSA(stack = "identity"),
-    "dig.dev.tooling" -> SSA("deploy"),
-    "dig.dev.tooling" -> SSA("domains"),
+    "digitalcms.dev" -> SSA("crosswords"),
     "digitalcms.dev" -> SSA("flexible"),
     "digitalcms.dev" -> SSA("flexible-secondary"),
     "digitalcms.dev" -> SSA("workflow"),
     "digitalcms.dev" -> SSA("cms-fronts"),
     "digitalcms.dev" -> SSA("elk-new"),
+    "digitalcms.dev" -> SSA("media-service"),
+    "digitalcms.dev" -> SSA("grid-elasticsearch"),
+    "digitalcms.dev" -> SSA("composer"),
+    "digital.investigations" -> SSA("pfi"),
     "digital.investigations" -> SSA("pfi-giant"),
     "digital.investigations" -> SSA("pfi-playground"),
     "digital.investigations" -> SSA("pfi-structured"),
-    "thegrid.dev" -> SSA("media-service"),
     "commercial.dev" -> SSA(stack = "frontend", app = Some("ipad-ad-preview")),
     "commercial.dev" -> SSA(stack = "flexible", app = Some("campaign-central")),
     "content.platforms" -> SSA(stack = "content-api"),
     "content.platforms" -> SSA(stack = "content-api-crier-v2"),
     "content.platforms" -> SSA(stack = "content-api-crier-v2-preview"),
     "content.platforms" -> SSA(stack = "content-api-dashboard"),
     "content.platforms" -> SSA(stack = "content-api-facebook-news-bot"),
+    "content.platforms" -> SSA(stack = "content-api-facebook-newstab"),
+    "content.platforms" -> SSA(stack = "content-api-fastly-cache-purger"),
+    "content.platforms" -> SSA(stack = "content-api-floodgate"),
```

Comment:
minor: wonder if there's benefit to supporting wildcarding, e.g. `content-api-*`?

Reply:
Yeah, I think that would definitely help to stop this list from drifting out of date so easily